.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should be in a .gitignore (as `/Users/jb` won't be a folder for most people)